confidential-data-hub/README.md

```diff
@@ -65,3 +65,16 @@ confidential-data-hub -c <path-to-config>
 ```
 
 Please see the [example config file](./example.config.toml) for more details.
+
+However, if a file isn't passed with **-c** then it will search for configurations on the
+following locations (in order):
+
+* **/etc/confidential-data-hub.toml**
+* **AA_KBC_PARAMS** environment variable
+* **agent.aa_kbc_params** parameter from the Kernel command-line (`/proc/cmdline`)
+
+There is a special case which is when running from [peer pods](http://github.com/confidential-containers/cloud-api-adaptor). It
```

Comment:
Should this be `https`?

Reply:
I will changed that and the other occurrences , thanks @Xynnn007 

The checker for links has failed on a relative link:
```
[ERR] [file:///home/runner/work/guest-components/guest-components/confidential-data-hub/docs/(../README.md#configuration-file)](file:///home/runner/work/guest-components/guest-components/confidential-data-hub/docs/(../README.md#configuration-file)) | Failed: Cannot find file
```

VSCode is able to follow the link...anyway, I will fix that.

Reply:
Fixed!